relayer-cli/src/utils/proof.ts

```diff
@@ -58,7 +60,7 @@ const getProofIndices = (nonce: number, count: number) => {
 
   const treeDepth = Math.ceil(Math.log2(count));
 
-  for (let i = 0; i < treeDepth; i++) {
+  for (let i = 0; i <= treeDepth; i++) {
```

Comment:
Are you 100% sure about this?

Reply:
log2(1) is 0 so in case of only 1 message it was failing to execute. turning into do while loop will fix that issue.